plugins/idea/pluginImpl/src/main/java/com/imaginea/kodebeagle/impl/action/RefreshAction.java

```diff
@@ -21,43 +21,57 @@
 import com.imaginea.kodebeagle.base.object.WindowObjects;
 import com.imaginea.kodebeagle.base.util.UIUtils;
 import com.imaginea.kodebeagle.impl.util.JavaImportsUtil;
-import com.imaginea.kodebeagle.impl.util.ScalaImportsUtil;
 import com.intellij.openapi.editor.Editor;
 
+// import com.imaginea.kodebeagle.impl.util.ScalaImportsUtil;
+
 public class RefreshAction extends RefreshActionBase {
-    private static final String FILETYPE_HELP_JAVA_SCALA = "<html><center>This plugin supports "
-            + "\"scala\" and \"java\" files only.</center></html>";
-    private static final String FILETYPE_HELP_JAVA = "<html><center>Plugin currently supports "
-            + "\"java\" files only.<br> To enable scala support install scala plugin!"
-            + "</center></html>";
-    private static final String FILE_EXT_SCALA = "scala";
-    private static final String FILE_EXT_SCALA_WORKSHEET = "sc";
+
+    private static final String FILETYPE_HELP_JAVA = "<html><center>This plugin supports "
+            + "\"java\" files only.</center></html>";
+
     private static final String FILE_EXT_JAVA = "java";
     private final WindowObjects windowObjects = WindowObjects.getInstance();
 
+    // TODO to be used after adding scala support
+    /* private static final String FILETYPE_HELP_JAVA_SCALA = "<html><center>This plugin supports "
+             + "\"scala\" and \"java\" files only.</center></html>";
+     private static final String FILETYPE_HELP_JAVA = "<html><center>Plugin currently supports "
+             + "\"java\" files only.<br> To enable scala support install scala plugin!"
+             + "</center></html>";*/
+    /*private static final String FILE_EXT_SCALA = "scala";
+    private static final String FILE_EXT_SCALA_WORKSHEET = "sc";*/
+
     @Override
     protected final void runAction() {
         final Editor editor = getEditor();
+
+        if (checkFileType(editor.getDocument(), FILE_EXT_JAVA)) {
+            windowObjects.setLanguage(FILE_EXT_JAVA);
+            doQuery(new JavaImportsUtil());
+        } else {
+            UIUtils uiUtils = new UIUtils();
+            uiUtils.showHelpInfo(FILETYPE_HELP_JAVA);
+        }
+
+        /*
+        // TODO to be used after adding scala support
+
         if (windowObjects.isOptionalDependencyPresent()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)